app/models/item.rb

```diff
@@ -9,7 +9,7 @@ class Item < ApplicationRecord
     validates :price, :presence => true, format: { with: /\A\d+(?:\.\d{0,2})?\z/ }, numericality: { greater_than: 0, less_than: 1000000 }
     validates :description, :presence => true, length: {maximum: 100}
     validate :image_type
-    #size_range: 1..3.megabytes
```

Comment:
why was this line deleted?

Reply:
It was a comment left in here a long time ago - we have no logic to check size of the image anyways